src/app/components/ElectronicDelivery/ElectronicDeliveryForm.jsx

```diff
@@ -210,31 +192,35 @@ class ElectronicDeliveryForm extends React.Component {
 
           <div className={`nypl-year-field ${errorClass['starting-page']}`}>
             <label htmlFor="starting-page" id="starting-page-label">Start Page
+              <span className="nypl-required-field">&nbsp;(Required)</span>
             </label>
             <input
-              id="starting-page"
+              id="start-page"
               type="number"
+              required
               className="form-text"
-              aria-labelledby="starting-page-label"
-              name="starting-page"
-              value={this.state.form['starting-page']}
-              onChange={(e) => this.handleUpdate(e, 'starting-page')}
+              aria-labelledby="start-page-label"
+              name="startPage"
+              value={this.state.form['start-page']}
```

Comment:
This is wrong. You updated the state to be `startPage`. It needs to coincide with the `name` attribute of the input element.